src/components/editor/markdown-renderer/replace-components/image/image-frame.tsx

```diff
@@ -1,7 +1,26 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import { useSelector } from 'react-redux'
+import { getProxiedUrl } from '../../../../../api/imageProxy'
+import { ApplicationState } from '../../../../../redux'
+
+export const ImageFrame: React.FC<React.ImgHTMLAttributes<HTMLImageElement>> = ({ alt, src, ...props }) => {
+  const [imageUrl, setImageUrl] = useState('')
+  const imageProxyEnabled = useSelector((state: ApplicationState) => state.backendConfig.imageProxy)
+
+  useEffect(() => {
+    if (!imageProxyEnabled) {
+      setImageUrl(src || '')
+      return
+    }
+    // This additional local function is needed because react-useEffect does not allow async functions as effect.
+    const fetchProxiedUrl = async (): Promise<void> => {
+      const proxiedUrlResponse = await getProxiedUrl(src || '')
```

Comment:
Why should we start an ajax call for an empty url?

Reply:
Done